launch-game.cmd

```diff
@@ -12,9 +12,9 @@ if "!ENGINE_VERSION!" == "" goto badconfig
 if "!ENGINE_DIRECTORY!" == "" goto badconfig
 
 set TEMPLATE_DIR=%CD%
-if not exist %ENGINE_DIRECTORY%\bin\OpenRA.exe goto noengine
->nul find %ENGINE_VERSION% %ENGINE_DIRECTORY%\VERSION || goto noengine
-cd %ENGINE_DIRECTORY%
+if not exist "%ENGINE_DIRECTORY%\bin\OpenRA.exe" goto noengine
+>nul find %ENGINE_VERSION% "%ENGINE_DIRECTORY%\VERSION" || goto noengine
```

Comment:
```suggestion
>nul find "%ENGINE_VERSION%" "%ENGINE_DIRECTORY%\VERSION" || goto noengine
```

Should fix the issue.

Reply:
Seems like it will treat the version string as a file name without the quotations, which lead to the error I had yesterday.